BlogTemplate/Pages/Post.cshtml.cs

```diff
@@ -10,7 +10,7 @@ namespace BlogTemplate._1.Pages
     public class PostModel : PageModel
     {
         private readonly BlogDataStore _dataStore;
-
+        public bool MaxedOutComments = false;

```

Comment:
This should be a property instead of a field.  It could be implemented as ` MaxedOutComents => Post.Comments.Count >= MaxAllowedComments`

Also, I would try to rename it to something with a more boolean-ish name, like CanAddComments, or IsCommentsFull or something like that.  "Maxed out" isn't as clear.